docs/course_team/concepts/course_overview_dashboard.rst

```diff
@@ -0,0 +1,43 @@
+Course Dashboard Reports
```

Comment:
The dashboard docs need to be in the reference section since the default Aspects settings link the "help" tab to them: https://docs.openedx.org/projects/openedx-aspects/page/reference/course_overview_dashboard.html